test/tests/functional/pbs_multi_sched.py

```diff
@@ -2079,6 +2079,14 @@ def degraded_resv_reconfirm(self, start, end, rrule=None, run=False):
         if rrule is not None:
             attr.update({ATTR_resv_rrule: rrule,
                          ATTR_resv_timezone: self.get_tzid()})
+        now = int(time.time())
+        if start <= now:
+            dur = end - start
+            start = now + 10
+            end = start + dur
+            attr['reserve_start'] = start
+            attr['reserve_end'] = end
```

Comment:
There is a copy of this function in the multisched test plan.  You should modify that one as well.